### PR TITLE
chore: make default public npm registry explicit

### PR DESCRIPTION
**Pull request by Knorgias**
## What I did

1. Make registry=https://registry.npmjs.org/ the default by adding it in .npmrc on project level
2. Update package-lock.json

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -22334,7 +22334,7 @@
     },
     "packages/ui": {
       "name": "@lion/ui",
-      "version": "0.1.5",
```
Comment (tkgroot) [inline package-lock.json:22337]: this change shouldn't be part of this PR, I assume.

**Review by tkgroot — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (Knorgias) [reply package-lock.json:22337]: Ran an `npm i` and quickly also updated the package-lock.
I also added it as (2) on the description above, so we can include this in this PR as well, if you agree.

Comment (tkgroot) [reply package-lock.json:22337]: sure, i dont know how strict you are with these things. Fine by me

**Review by tkgroot — APPROVED** [review]

**Review by tlouisse — APPROVED** [review]
